src/alert_suppressions_runner.py

```diff
@@ -100,6 +100,8 @@ def main():
     for squelch_name in load_rules(ctx, ALERT_SQUELCH_POSTFIX):
         run_suppressions(squelch_name)
     flag_remaining_alerts(ctx)
+    if {CLOUDWATCH_METRICS}:
```

Comment:
in Python, this is a non-empty set, which will always evaluate to Truthy. if you skip the `{}` it'll be what you prob intend